client/gutenberg/extensions/mailchimp/view.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * Internal dependencies
+ */
+import './view.scss';
+
+const blockClassName = 'wp-block-jetpack-mailchimp';
+
+const JetpackEmailSubscribe = {
```

Comment:
Is there a reason to group these functions under an object that's not a `class`? Could they just be top-level `function`s in this module?

Reply:
Sounds fine.